tests/hud_test.py

```diff
@@ -0,0 +1,57 @@
+from time import sleep
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def setup(mocker):
+    mocker.patch("pt_miniscreen.pages.system.login.getuser", return_value="root")
+    mocker.patch(
+        "pt_miniscreen.pages.system.login.is_pi_using_default_password",
+        return_value=True,
+    )
+
+
+def test_hud_navigation(miniscreen, snapshot):
+    sleep(1)
```

Comment:
can we replace these `sleep` (in all test-files) with `Event.wait()`?

Reply:
I'm not sure why we would use event.wait. Something would need to set the event, otherwise if we use a timeout it is effector the same as a sleep.

Reply:
sooo those `sleep` are meant to wait for the transition to finish, right? if we `wait` for the 'transition done' event (we used to have one, not sure anymore), we could use them to replace the sleeps

Reply:
Right I see what you are saying! Yeah the are to wait for transitions to end. I'm not super keen on hooking into the apps event system for that, these tests are meant to be as decoupled from the implementation as possible so internal changes can be made without the tests being broken.

As you can imagine that hasn't been as possible as I would have liked! But yeah that would move us in the wrong direction I think. Hopefully in future we can figure out how to freeze and step time in a more controlled way like you mentioned earlier which would make this better!